cryptofeed/deribit/deribit.py

```diff
@@ -0,0 +1,205 @@
+import logging
+import yaml
+import json
+import requests
+
+from cryptofeed.feed import Feed
+from cryptofeed.defines import DERIBIT, BUY, SELL, TRADES, BID, ASK, TICKER, L2_BOOK
+from cryptofeed.standards import pair_exchange_to_std, timestamp_normalize
+
+from sortedcontainers import SortedDict as sd
+from decimal import Decimal
+
+LOG = logging.getLogger('feedhandler')
+
+
+class Deribit(Feed):
+    id = DERIBIT
+
+    def __init__(self, pairs=None, channels=None, callbacks=None, config=None, **kwargs):
+        super().__init__('wss://test.deribit.com/ws/api/v2', pairs=pairs,
```

Comment:
thats the testnet endpoint right? not the live endpoint?

Reply:
Yes, that's the test endpoint

Reply:
@vivikar can we put in the correct endpoint for live data?

Reply:
We can't. There is no live endpoint for Deribit api v2. As far as I understand api v2 is still under development, so they have only test endpoint. However the data we get is coming live, the only difference is that we get it through their test api.

Reply:
i tested your code with wss://www.deribit.com/ws/api/v2 and it works, so I think it does exist, it may not be publicized though